### DISCUSSION TITLE
How can I run kitty on my server (ubuntu server)?

### DISCUSSION THREAD

**Discussion by rcv11x** [kovidgoyal/kitty#7336] [Q&A]
Good, I've been using kitty for 4 months on desktop and everything great.

The problem I have now is that I use a proxmox server and a LXC which is like a virtual machine (with ubuntu server 22.04.4) without GUI and I have installed kitty like this: `curl -L https://sw.kovidgoyal.net/kitty/installer.sh | sh /dev/stdin` but when I run kitty like this: `/root/.local/kitty.app/bin/kitty` I get the following error:

![2024-04-11_14-01kitty](https://github.com/kovidgoyal/kitty/assets/38696644/837cedb4-a32e-4ae2-baab-e3b86cffb62e)

how can I solve it or if it is possible, if it is not possible I would have to use tmux to manage my servers but I like kitty much better.

**Comment by kovidgoyal** [comment]
kitty is a graphical program you dont run it on your server. You run kitty on your desktop and use ssh to connect to your server. If you use the ssh kitten in kitty you get lots of goodies like  persisten connection, automatic creation of new windows pre logged in to the server etc.
